package_seed_lists/effortless_x86_64-linux_stable

```diff
@@ -1,4 +1,5 @@
 chef/chef-client
```

Comment:
I'm confused about us having both chef/chef-client and chef/chef-infra-client; are they both being used?

Reply:
chef/chef-client is chef 14 and chef/chef-infra-client is chef 15 and will be chef 16 when it's released.

Reply:
Ah, ok, makes sense.